*** rollup.config.js	2021-11-19 06:34:40.000000000 -0600
--- fix-rollup.config	2021-12-22 16:51:49.037220718 -0600
***************
*** 4,9 ****
--- 4,10 ----
  import livereload from 'rollup-plugin-livereload';
  import { terser } from 'rollup-plugin-terser';
  import css from 'rollup-plugin-css-only';
+ import replace from '@rollup/plugin-replace';
  
  const production = !process.env.ROLLUP_WATCH;
  
***************
*** 68,74 ****
  
  		// If we're building for production (npm run build
  		// instead of npm run dev), minify
! 		production && terser()
  	],
  	watch: {
  		clearScreen: false
--- 69,79 ----
  
  		// If we're building for production (npm run build
  		// instead of npm run dev), minify
! 		production && terser(),
! 
!     replace({
!       "process.env.NODE_ENV": JSON.stringify("development"),
!     }),
  	],
  	watch: {
  		clearScreen: false
